src/main/java/se/sundsvall/parkingpermit/businesslogic/worker/investigation/ExecuteRulesTaskWorker.java

```diff
@@ -32,7 +33,8 @@ protected void executeBusinessLogic(ExternalTask externalTask, ExternalTaskServi
 		try {
 			logInfo("Execute Worker for ExecuteRulesTaskWorker");
 			final var errand = getErrand(externalTask);
-			final var ruleEngineResponse = businessRulesClient.runRuleEngine(toRuleEngineRequest(errand));
+			final String municipalityId = externalTask.getVariable(CAMUNDA_VARIABLE_MUNICIPALITY_ID);
```

Comment:
I guess it's a `String` because of reasons?

Reply:
Yes, `getVariable` are using generics, so the return type must be specified.